testinfra/development/test_development_application_settings.py

```diff
@@ -120,12 +127,12 @@ def test_development_default_logo_exists(File):
     TODO: Add check for custom logo file.
     """
 
-    f = File("/vagrant/securedrop/static/i/logo.png")
+    f = File("{}/static/i/logo.png".format(sd_test_vars.securedrop_code))
     assert f.is_file
-    assert f.user == "vagrant"
-    assert f.group == "vagrant"
-    assert oct(f.mode) == "0644"
-    # TODO: Ansible task declares mode 400 but not as string, needs to be fixed
-    # and tests updated. Also, not using "mode" in tests below because umask
-    # on snapci machines differs from the /vagrant folder in dev VM.
-    # Fixing Ansible task may fix differing perms.
+    assert f.user == sd_test_vars.securedrop_user
+    assert f.group == sd_test_vars.securedrop_user
+    # check if logo is the default securedrop png
+    if f.md5sum == "92443946d5c9e05020a090f97b62d027":
+        assert oct(f.mode) == "0664"
```

Comment:
Different umask in both environments most likely. Agreed on punt the check to a separate issue; we'll readd the check once the first-run config actually enforces it.